docs/source/changelog.rst

```diff
@@ -475,7 +475,7 @@ Version 0.8.3
 Features:
 
 - error options for cat
-- memory fs created time in detailed `ls`
+- memory fs created time in detailed ``ls```
```

Comment:
Single backticks means a sphinx link, so where there is no such target double-backticks are used to that the contained text is rendered as code. So single backticks in a github comment are equivalent to double backticks in sphinx documentation.